auxx/src/Mode.hs

```diff
@@ -216,3 +219,9 @@ instance MonadFormatPeers AuxxMode where
 instance HasConfiguration => MonadAddresses AuxxMode where
     type AddrData AuxxMode = PublicKey
     getNewAddress = makePubKeyAddressAuxx
+
+type instance MempoolExt AuxxMode = EmptyMempoolExt
+
+instance (HasConfiguration, HasInfraConfiguration) => MonadTxpLocal AuxxMode where
+    txpNormalize = txNormalize
```

Comment:
why?

Reply:
Because look at how all other instances are defined. They all use `realModeToAuxx`. I see you changed it to use `withReaderT acRealModeContext` already, it would be better to use `realModeToAuxx` (which is the same), but it's minor.